### ISSUE TITLE
is this still maintained?

### ISSUE THREAD

**Issue by stephenmathieson** [codevise/jenkins-nvm-plugin#3]
searching for [nvm](https://wiki.jenkins-ci.org/dosearchsite.action?queryString=nvm) returns no results...

**Comment by tf** [comment]
At least, we use it everyday. I never pushed the plugin to the official jenkins plugin registry. But it can be installed via jpi. In @ampledata's fork there is a Makefile. Maybe that can help.

**Comment by stephenmathieson** [comment]
looks like the fork has been deleted.

i'll try to find some time to get this installed (and portable).  would you take a pr with build/installation steps?

for the time being, i've just got an _execute shell_ build step, which sources `~/.nvm/nvm.sh` before each build.  it's kinda awkward, but it works just fine.

**Comment by tf** [comment]
Yes, that would be great. I'd also be open to pushing the plugin to the public index. It's just that I have never done that before.

**Comment by ampledata** [comment]
our fork is still there...? That Makefile and updated Gemfile should work, at least that's what we're using locally to build this plugin. Feel free to lift all those changes and put them into a legit PR.

**Comment by stephenmathieson** [comment]
maybe i'm missing something, but https://github.com/ampledata/jenkins-nvm-plugin 404s...

**Comment by ampledata** [comment]
Oh check out the Network map for the plugin on GitHub, my fork is listed under GitHub.com/OnBeep--Greg AlbrechtFounder & CTO, OnBeep, Inc.gba@onbeep.com310-621-9598@ampledataOn Thu, May 15, 2014 at 6:22 AM -0700, "Stephen Mathieson" notifications@github.com wrote:

maybe i'm missing something, but https://github.com/ampledata/jenkins-nvm-plugin 404s...

**Comment by ampledata** [comment]
https://github.com/OnBeep/jenkins-nvm-plugin/tree/feature/ampledata-updates
